expansion/src/main/java/no/unit/nva/expansion/utils/ExpandedTicketStatusQueries.java

```diff
@@ -0,0 +1,40 @@
+package no.unit.nva.expansion.utils;
+
+import no.unit.nva.expansion.model.ExpandedTicketStatus;
+import no.unit.nva.publication.model.business.TicketEntry;
+import no.unit.nva.publication.model.business.TicketStatus;
+
+import java.util.Objects;
+
+public class ExpandedTicketStatusQueries {
+    public static ExpandedTicketStatus getExpandedTicketStatus(TicketEntry ticketEntry) {
```

Comment:
Do you need a switch here? I think this could be replaced with the ExpandedTicketStatus.valueOf() method:
```
public static ExpandedTicketStatus getExpandedTicketStatus(TicketEntry ticketEntry) {
        if (isPending(ticketEntry)) {
            return getNewTicketStatus(ticketEntry);
        }
        return ExpandedTicketStatus.valueOf(ticketEntry.getStatusString());
    }

    public static TicketStatus getTicketStatus(ExpandedTicketStatus expandedTicketStatus) {
        if (expandedTicketStatus.equals(ExpandedTicketStatus.NEW)) {
            return TicketStatus.PENDING;
        } else {
            return TicketStatus.parse(expandedTicketStatus.toString());
        }
    }

    static ExpandedTicketStatus getNewTicketStatus(TicketEntry ticketEntry) {
        if (isNull(ticketEntry.getAssignee())) {
            return ExpandedTicketStatus.NEW;
        } else {
            return ExpandedTicketStatus.PENDING;
        }
    }

    private static boolean isPending(TicketEntry ticketEntry) {
        return TicketStatus.PENDING.equals(ticketEntry.getStatus());
    }
```